easybuild/easyblocks/t/tensorflow.py

```diff
@@ -209,7 +209,6 @@ def configure_step(self):
             'TF_NEED_OPENCL_SYCL': '0',
             'TF_NEED_S3': '0',  # Amazon S3 File System
             'TF_NEED_VERBS': '0',
-            'TF_NEED_TENSORRT': ('0', '1')[bool(tensorrt_root)],
```

Comment:
@akesandgren Better set to `0` by default, to avoid surprises?

Reply:
Done